modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/RichText/RichText.es.js

```diff
@@ -193,6 +192,7 @@ const RichText = ({
 						}}
 						readOnly={readOnly}
 						ref={editorRef}
+						required={otherProps.required}
```

Comment:
This is generally a good approach. `...otherProps` mechanism in `ClassicEditor` is quite misleading :disappointed:  The idea is typically to forward props the most relevant element in the component, but in this case, only a small subset ends up [applied](https://github.com/ckeditor/ckeditor4-react/blob/v1.4.2/src/ckeditor.jsx). Having it is a way of future-proofing, but I'm not sure if it's worth the clarity hit.

Reply:
I would avoid naming this `required` as it suggests `<ClassicEditor>` will enforce some form of mandatoryness. As this is not the case , and the sole purpose of the `required` prop is to handle aria markup, prop should also be named `aria-required` to avoid confusion. 

It corresponds to upper layers (FieldBase, RichText) to handle the mandatoryness checks.